projects/ngx-qgrid-plugins/src/lib/cell-tooltip/cell-tooltip.directive.ts

```diff
@@ -0,0 +1,39 @@
+import {
+	OnChanges,
+	Input,
+	Directive,
+	ElementRef,
+	SimpleChanges,
+} from '@angular/core';
+import { GridError, GridPlugin, TemplateHostService } from '@qgrid/ngx';
+import { jobLine } from '@qgrid/core/services/job.line';
+@Directive({
+	selector: '[q-grid-tooltip]',
+	providers: [GridPlugin, TemplateHostService],
+})
+export class CellTooltipDirective implements OnChanges {
+	@Input() showDelay = 1000;
+	@Input() host: HTMLElement;
+	private job = jobLine(this.showDelay);
```

Comment:
private to the top, could you please check linter why it doesn't require it?

Reply:
All other corrections has been made, except this one.

It's not possible becouse we can not use property  'showDelay'  before its initialization.ts(2729)

Reply:
add new line at least

Reply:
added